package.json

```diff
@@ -62,12 +69,18 @@
     "rollup-plugin-terser": "^7.0.2",
     "semver": "^7.3.5",
     "ts-jest": "^26.5.6",
+    "type-fest": "^2.13.0",
     "typescript": "^4.3.5"
   },
+  "resolutions": {
```

Comment:
The `react-native` types declare a dependency on these packages without a version (`*`) so this prevents us from including React 18 types if the lockfile gets borked.